packages/react-liveness/src/components/FaceLivenessDetector/shared/FaceLivenessErrorModal.tsx

```diff
@@ -41,34 +42,40 @@ const renderToastErrorModal = (props: {
 
   let heading: string;
   let message: string;
+  let label: string;
 
   switch (errorState) {
     case LivenessErrorState.TIMEOUT:
       heading = timeoutHeaderText;
       message = timeoutMessageText;
+      label = errorLabelText;
```

Comment:
small nit: if it's the same in all cases we can set the value outside of the switch statement right?

Reply:
+1